docs/_providers/internetbs.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Internet.bs
+title: Internet.bs Provider
+layout: default
+jsId: INTERNETBS
+---
+# Internet.bs Provider
+
+Internet.bs has API to mange your DNS zones. But in current version of provider, it's not implemented. You can use Internet.bs only as registrar for now.
```

Comment:
Suggestion:

DNSControl's Internet.bs provider supports being a Registrar.  Support for being a DNS Provider is not included, but could be added in the future.

Reply:
agree